### ISSUE TITLE
A questions about the boundary loss implementation.

### ISSUE THREAD

**Issue by Hung-Hsiang-617** [JunMa11/SegWithDistMap#13]
Hi,
I do not understand the code equal paper function

multipled = torch.einsum('bxyz, bxyz->bxyz', pc, dc)
bd_loss = multipled.mean()

![image](https://user-images.githubusercontent.com/53552921/109634097-cfabb100-7b83-11eb-819b-6adc9e18e8b1.png)

**Comment by JunMa11** [comment]
Hi @Hung-Hsiang-617,

I apologize for the very late reply. Recently, I'm busy with the [FLARE21 Challenge](https://flare.grand-challenge.org/).

I would recommend checking the original implementation and torch.einsum
https://github.com/LIVIAETS/boundary-loss/blob/749602d822fcb097c79a7fb708805b8982c37030/losses.py#L76
